### ISSUE TITLE
[Question]: I can't start the server

### ISSUE THREAD

**Issue by trzepiz** [IBMa/equal-access#2209]
### Project

accessibility-checker for Node

### Description

Node: v22.14.0
npm - 9.8.1

I do everything with https://github.com/IBMa/equal-access?tab=readme-ov-file#build

D:\NODE\equal-access\rule-server> npm start

```
webpack 5.98.0 compiled with 3 warnings in 5594 ms

> accessibility-checker-engine@3.0.0 build:help
> ts-node src/genHelp.ts

[14:58:35] Working directory changed to D:\NODE\equal-access\rule-server\gulp
[14:58:35] Using gulpfile D:\NODE\equal-access\rule-server\gulp\gulpfile.js
[14:58:35] Starting 'build'...
[14:58:35] Starting 'distFiles'...
[14:58:35] Starting 'staticFiles'...
[14:58:35] Starting 'acePreview'...
[14:58:35] Starting 'aceDocs'...
[14:58:35] Starting 'archivePolicies'...
[14:58:38] Finished 'distFiles' after 2.77 s
[14:58:38] Finished 'archivePolicies' after 2.76 s
[14:58:38] Finished 'acePreview' after 2.79 s
[14:58:38] Finished 'aceDocs' after 2.99 s
[14:58:42] Finished 'staticFiles' after 7.44 s
[14:58:42] Finished 'build' after 7.45 s
[INFO] Endpoint: https://localhost:9445
Latest: D:\NODE\equal-access\rule-server\dist\archives\2025.02.10
[Uncaught] AggregateError:
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1712:7) {
  scope: 'socket',
  errid: 'request',
  code: 'ECONNREFUSED',
  response: undefined,
  stacktrace: [
    'AggregateError: ',
    '    at internalConnectMultiple (node:net:1139:18)',
    '    at afterConnectMultiple (node:net:1712:7)'
  ]
}
Error: Could not find openssl on your system on this path: openssl
    at D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\openssl.js:258:1
[Uncaught] TypeError: Cannot read properties of undefined (reading 'certificate')
    at callback (D:\NODE\equal-access\rule-server\dist\main.js:129:84)
    at callback (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\pem.js:362:1)
    at callback (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\pem.js:240:1)
    at done (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\pem.js:97:1)
    at finalCallback (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\pem.js:111:1)
    at deleteSeries (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\helper.js:130:1)
    at Object.deleteTempFiles (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\helper.js:133:1)
    at callback (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\pem.js:104:1)
    at callback (D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\openssl.js:70:1)
    at D:\NODE\equal-access\rule-server\node_modules\pem\webpack:\pem\lib\openssl.js:243:1
PS D:\NODE\equal-access\rule-server>
```

**Comment by trzepiz** [comment]
I run /local.sh in /rule-server and server has started. 
But localhost:9445 give me **"Cannot GET /"**
```
[15:29:12] Working directory changed to D:\NODE\equal-access\rule-server\gulp
[15:29:12] Using gulpfile D:\NODE\equal-access\rule-server\gulp\gulpfile.js
[15:29:12] Starting 'build'...
[15:29:12] Starting 'distFiles'...
[15:29:12] Starting 'staticFiles'...
[15:29:12] Starting 'acePreview'...
[15:29:12] Starting 'aceDocs'...
[15:29:12] Starting 'archivePolicies'...
[15:29:16] Finished 'distFiles' after 3.65 s
[15:29:16] Finished 'archivePolicies' after 3.65 s
[15:29:16] Finished 'acePreview' after 3.7 s
[15:29:16] Finished 'aceDocs' after 3.9 s
[15:29:22] Finished 'staticFiles' after 9.41 s
[15:29:22] Finished 'build' after 9.41 s
[INFO] Endpoint: https://localhost:9445
Latest: D:\NODE\equal-access\rule-server\dist\archives\2025.02.10
[Uncaught] AggregateError:
    at internalConnectMultiple (node:net:1139:18)
    at afterConnectMultiple (node:net:1712:7) {
  scope: 'socket',
  errid: 'request',
  code: 'ECONNREFUSED',
  response: undefined,
  stacktrace: [
    'AggregateError: ',
    '    at internalConnectMultiple (node:net:1139:18)',
    '    at afterConnectMultiple (node:net:1712:7)'
  ]
}
{ app: 'rule-server', url: 'https://localhost:9445' } App started (cert generated)
```

**Comment by tombrunet** [comment]
Looks like you're missing some things with your OS setup. But, you're trying to run development processes, which we don't support. If you have an issue with the tools themselves, let us know!

**Comment by tombrunet** [comment]
As an aside, we mostly develop on Mac, but our automated builds run on both Linux and Windows. openssl does not come installed by default on Windows, so looks like you need to find packages to support that.
